### DISCUSSION TITLE
Game end statistics

### DISCUSSION THREAD

**Discussion by Stiivi** [ihhub/fheroes2#7578] [Ideas]
### Preliminary checks

- [X] I've checked that there aren't [**other open issues**](https://github.com/ihhub/fheroes2/issues?q=is%3Aissue) on the same topic.

### Describe the problem requiring a solution

This is an improvement for victory enjoyment. When I finish the game, I would like to see more detailed statistics. Some maps are not tuned to the built-in high-score rating based purely on days. Moreover, sometimes I would like to play a map for leisure only focusing on building my kingdom (holding the last hero hostage in his castle, while building/exploring/capturing the rest of the map). Finishing a more complex map or playing leisure-style shows a simple victory notice and that is it. Does not feel very satisfying.

### Describe the possible solution

It would be nice if there was an additional simple stats displayed in/after/before the victory window:

Hero development:
- Highest hero level (highest str/def/power/int) (*)
- Number of Artefacts gathered

Kingdom development:
- Completeness: how many buildings (out of how many) were built
- How many cities are 100% complete (*)
- Number of top-tier creatures bought

Battles:
- Battles won, lost or abandoned or maybe a ratio: (battles won / total battles) (*)
- Total money lost retreating from a battle
- Creatures lost or killed

Not to overwhelm the player with lots of stats, I marked three with (*) that might be displayed in the "end of game" dialogue, maybe even recorded somewhere for further comparison. The rest might be displayed after clicking "advanced". Days may me displayed too.

Inspiration from Zachtronics games, such as Opus Magnum: Cost/Cycles/Area or Exapunks: Cycles/Size/Activity.

**Comment by ihhub** [comment]
Hi @Stiivi , even if we implement them where exactly this data should be displayed? Highscores as well as victory animation board have no space for extra text.

**Comment by ihhub** [comment]
I am moving this issue into the discussion section as it requires heavy work on UI which has not been proposed at the moment.

**Comment by Mr-Bajs** [comment]
Some more data would be intresting to have.
Some simple data such as date when the map was completed, what version of the game was played on completion and starting class. An right click infobox would suffice.
